src/components/modal.ts

```diff
@@ -114,6 +114,7 @@ export default class Modal {
 
         if (this.current) {
             this.current.modal('hide');
+            this.current.remove();
```

Comment:
There was a bug related to the Modal, I forgot to use this code and after closing the modal, it still existed in the DOM.